Formula/p/pyqt-builder.rb

```diff
@@ -17,17 +20,33 @@ class PyqtBuilder < Formula
   end
 
   depends_on "python@3.12"
-  depends_on "sip"
 
   def python3
     "python3.12"
   end
 
+  resource "packaging" do
+    url "https://files.pythonhosted.org/packages/ee/b5/b43a27ac7472e1818c4bafd44430e69605baefe1f34440593e0332ec8b4d/packaging-24.0.tar.gz"
+    sha256 "eb82c5e3e56209074766e6885bb04b8c38a0c015d0a30036ebe7ece34c9989e9"
+  end
+
+  resource "setuptools" do
+    url "https://files.pythonhosted.org/packages/c8/1f/e026746e5885a83e1af99002ae63650b7c577af5c424d4c27edcf729ab44/setuptools-69.1.1.tar.gz"
+    sha256 "5c0806c7d9af348e6dd3777b4f4dbb42c7ad85b190104837488eab9a7c945cf8"
+  end
+
+  resource "sip" do
+    url "https://files.pythonhosted.org/packages/99/85/261c41cc709f65d5b87669f42e502d05cc544c24884121bc594ab0329d8e/sip-6.8.3.tar.gz"
+    sha256 "888547b018bb24c36aded519e93d3e513d4c6aa0ba55b7cc1affbd45cf10762c"
+  end
+
   def install
-    system python3, "-m", "pip", "install", *std_pip_args, "."
+    virtualenv_install_with_resources
   end
 
   test do
+    ENV.prepend_path "PYTHONPATH", libexec/Language::Python.site_packages(python3)
+
     system bin/"pyqt-bundle", "-V"
     system python3, "-c", "import pyqtbuild"
```

Comment:
I just mean this:

```suggestion
    system bin/"pyqt-bundle", "-V"
```

Reply:
then that means the import wont work at all?

Reply:
right, and from the user's perspective it also won't work / be importable (since a venv is isolated)

I'm just saying the point of this test was that a user could `brew install pyqt-builder` and then be able to `import pyqtbuild` without configuring anything. Since this PR removes that functionality, we should remove the test. Adding a test-only `PYTHONPATH` is just cheating since a user isn't going to do that

Reply:
I see

Reply:
Without that functionality, does it make sense for Homebrew to even ship this?

Reply:
Need to think about this some more as our main usage was for building `pyqt` but these changes will probably break that, i.e. `project.py` does `from pyqtbuild ...`.

Have to check if we should build in isolated venv instead. Only thing is the extra inputs we pass via `sip-install`
https://github.com/Homebrew/homebrew-core/blob/cd31c9956bd24474f7d4d2f2bf697f4a042e70b5/Formula/p/pyqt.rb#L65-L70